sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutput.java

```diff
@@ -7,9 +7,9 @@
 Software distributed under the License is distributed on an "AS IS" basis,
 WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License
 for the specific language governing rights and limitations under the License.
- 
```

Comment:
I checked the file out from the last commit you made. I think the IDE still just automatically stripped some of the whitespace characters out.

Reply:
I think you should disable auto-formatting in your IDE when you modify existing code (especially if it has different formatting rules). I used to have it enabled too but I realized it makes it really hard to look at commit diffs because the important changes are lost in the middle of zillions of formatting changes.

You can leave it enabled when you create your own code but then disable it when you modify existing code.